packages/google_maps_flutter/google_maps_flutter_android/example/pubspec.yaml

```diff
@@ -32,3 +32,8 @@ flutter:
   uses-material-design: true
   assets:
     - assets/
+
+# FOR TESTING AND INITIAL REVIEW ONLY. DO NOT MERGE.
```

Comment:
Please revert the changes to `google_maps_flutter_android` and `google_maps_flutter_ios`.